lib/controllers/interpret.dart

```diff
@@ -77,8 +79,9 @@ class Interpret {
         var tempTree = currentScreen;
         tempTree.name = designPage.name;
 
+        tempTree.data = PBGenerationViewData();
         if (currentScreen.rootNode is InheritedScaffold) {
-          tempTree.tree_type = TREE_TYPE.SCREEN;
+          PBPlatformOrientationLinkerService().addOrientationPlatformInformation(tempTree);
```

Comment:
I think we remove `tempTree.tree_type = TREE_TYPE.SCREEN;` by mistake
that should still be part of this